collector/toc.yml

```diff
@@ -8,3 +8,15 @@
     href: collector/settings
   - name: Upgrading
     href: collector/upgrading
+  - name: OpenTelemetry Setup
+    href: collector/otel-setup
+- mount: collector/otel-setup
+  items:
+  - name: Overview
+    href: collector/otel-setup
+  - name: Set up traceparent query tag
+    href: collector/otel-setup/traceparent
+  - name: OpenTelemetry Setup with Honeycomb
+    href: collector/otel-setup/honeycomb
+  - name: OpenTelemetry Setup with New Relic
+    href: collector/otel-setup/newrelic
```

Comment:
I was thinking we actually put this at the top-level, to highlight the fact that we integrate with APM providers more explicitly (its a bit hard to find in the collector sub-navigation).

Like this, also emphasizing integrating for the New Relic / Honeycomb pages:

<img width="370" alt="Screenshot 2023-10-24 at 11 03 29 PM" src="https://github.com/pganalyze/pganalyze-docs/assets/7227/ed7c5c32-d2dd-4e7f-9403-96ba3cee14b0">

Reply:
I've updated the structure, can you please double check and see if this is what you're looking for?

https://pganalyze-we-docs-updat-lyhnyi.herokuapp.com/docs/opentelemetry

Reply:
Yes, that is exactly what I was thinking 👍